axelor-helpdesk/src/main/resources/views/Charts.xml

```diff
@@ -41,18 +41,23 @@
       translation.message_value
       END) as _status
       FROM
-      helpdesk_ticket
+      helpdesk_ticket AS
+      _ticket
       LEFT
       JOIN
       meta_translation as translation
       ON
       translation.message_key = 'chart.resolved' AND
       translation.language = :_language
+      LEFT JOIN
+      helpdesk_ticket_status AS _ticket_status
+      ON
+      _ticket.ticket_status = _ticket_status.id
       WHERE
-      DATE(created_on) BETWEEN DATE(:fromDate) AND
-      DATE(:toDate)
-      AND assigned_to_user = :_user_id
-      AND status_select = 2
+      DATE(_ticket.created_on) BETWEEN DATE(:fromDate)
+      AND DATE(:toDate)
+      AND _ticket.assigned_to_user = :_user_id
+      AND _ticket_status.name = 'Resolved'
```

Comment:
Resolved status is not determined by its name but in AppHeldesk  along with default, closed, ongoing status